CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - fix: Set description for performance tracker spans (#1042)
 - fix: Capturing Child Spans (#1139)
 - feat: Auto UI Performance Instrumentation (#1105)
+- ref: SentryPerformanceTracker (#1144)
```

Comment:
`l`: We only put feat and fix in the changelog. You can use `#skip-changelog` in the description of the PR to make danger happy.
```suggestion
```